cmd/remote/cmd.go

```diff
@@ -26,7 +25,7 @@ func main() {
 func realMain() error {
 
 	ctx := context.Background()
-	logger := golog.NewDevelopmentLogger("client")
+	logger := logging.NewLogger("client")
```

Comment:
The module would not build locally unless I changed the logger to the new  style logger in the rdk. runWebWithConfig needed a `logging.Logger` for me. Can rever if this is not the case on whatever this is being built on.

Reply:
Changing to `logging.NewLogger` will no longer log debug-level logs (which the previous logger did).

From a cursory glance, the only code in this repo that logs at debug level is here: https://github.com/erh/viamrtsp/blob/main/rtsp.go#L128

Could I recommend changing this to:
```suggestion
	logger := module.NewLoggerFromArgs("client")
```

That way module will default to info level logs but allow the end user to configure the level at runtime: https://pkg.go.dev/go.viam.com/rdk@v0.23.2/module#NewLoggerFromArgs

Reply:
Thanks! I'll make the change.